storage/options.go

```diff
@@ -67,6 +67,9 @@ const (
 
 	// defaultIndexingEnabled disables indexing by default
 	defaultIndexingEnabled = false
+
+	// defaultMinSnapshotInterval is the default minimum interval that must elapse between snapshots
+	defaultMinSnapshotInterval = time.Minute
```

Comment:
hm this is aggressive. could we bump up to something in the order of 10 minutes?

Reply:
I don't think its particularly aggressive, lets discuss in person

Reply:
Yeah we should let this match the min tick interval which is 1min.